File: osis/labs/LR-5-6/block-1/main.pas
program RandomArraySum;

var
  a, b: array[1..10] of Integer;
  i, a_sum, b_sum: Integer;
begin
  { Заполняем массивы случайными числами от 1 до 100 }
  for i := 1 to 10 do
  begin
    a[i] := Random(100) + 1;
    b[i] := Random(100) + 1;
  end;

  Writeln(a);
  Writeln(b);
  
  a_sum := 0;
  b_sum := 0;
  
  for i := 1 to 10 do
  begin
    if a[i] > 0 then
      a_sum := a_sum + a[i];
    if b[i] > 0 then
      b_sum := b_sum + b[i];
  end;

  if a_sum > b_sum then
  begin
    for i := 1 to 10 do
      b[i] := b[i] * 10;
      Writeln(b);
  end
  else
  begin
    for i := 1 to 10 do
      a[i] := a[i] * 10;
      Writeln(a);
  end;
end.

File: osis/labs/LR-5-6/block-1/main.py
from random import randint

a = [randint(1, 100) for _ in range(10)]
b = [randint(1, 100) for _ in range(10)]
print(a, b, sep="\n")

a_sum = 0
b_sum = 0
for i in a:
    if i > 0:
        a_sum += i
    
for i in b:
    if i > 0:
        b_sum += i
        
if a_sum > b_sum:
    for i in range(len(b)-1):
        b[i] = b[i] * 10
    print(b)
else:
    for i in range(len(a)-1):
        a[i] = a[i] * 10
    print(a)